mantle/kola/tests/misc/network.go

```diff
@@ -300,10 +300,10 @@ func NetworkSecondaryNics(c cluster.TestCluster) {
 
 	setupMultipleNetworkTest(c, primaryMac, secondaryMac)
 
-	checkOvsBridge(c, primaryMac)
+	checkOvsBridge(c, primaryMac, "")
 }
 
-func checkOvsBridge(c cluster.TestCluster, setPrimaryMac string) {
+func checkOvsBridge(c cluster.TestCluster, setPrimaryMac, setPrimaryIp string) {
 	var err error
 	machineIdx := 0
```

Comment:
Not new, but I don't think we need this, right? We only provision one machine so the right one is always going to be 0. Then you can avoid needing a parameter for that in `getBondIfaceIP` (or if you want to keep it generic, passing a `platform.Machine` would be a bit cleaner).

Reply:
totally agree. Done